src/inmanta/agent/executor.py

```diff
@@ -91,6 +91,10 @@ class EnvBlueprint:
     requirements: Sequence[str]
     _hash_cache: Optional[str] = dataclasses.field(default=None, init=False, repr=False)
 
+    def __post_init__(self) -> None:
+        # remove duplicates and make uniform
+        self.requirements = sorted(list(set(self.requirements)))
```

Comment:
No need for the `list` in between I think.
```suggestion
        self.requirements = sorted(set(self.requirements))
```